client/hawc_client/__init__.py

```diff
@@ -191,6 +199,21 @@ def full_data(self, assessment_id: int) -> pd.DataFrame:
         response_json = self.session.get(url)
         return pd.DataFrame(response_json)
 
+    def create(
```

Comment:
done and will be in the next push. I also modified the rob score override serializer to make the get_object_url/get_object_name fields read-only since those just get generated on the fly based on the actual content object of an override.